pyccel/codegen/pipeline.py

```diff
@@ -174,6 +179,46 @@ def handle_error(stage):
         parsers = [parser]
         module_names = [module_name]
 
+    # -------------------------------------------------------------------------
+    internal_lib_dict = {}
+
+    def get_internal_lib(import_lib):
+        # get path to pyccel/stdlib/import_lib
+        import pyccel.stdlib as stdlib_folder
+        stdlib_path = os.path.dirname(stdlib_folder.__file__)
+        lib_path = os.path.join(stdlib_path, import_lib)
+
+        # remove library folder from pyccel if exist then copy new one
+        lib_dest_path = os.path.join(pyccel_dirpath, import_lib)
+        if os.path.exists(lib_dest_path):
+            shutil.rmtree(lib_dest_path)
+        shutil.copytree(lib_path, lib_dest_path)
+        # get all c files in library directory
+        c_files = []
+        for e in os.listdir(lib_dest_path):
+            if e.endswith(".c"):
+                c_files.append(os.path.join(lib_dest_path, e))
```

Comment:
yes, just more works.
do you need me to do it also for fortran?

Reply:
How much more work is it? It looks like just an `or` to me

Reply:
there are some other changes to make but not a lot. if all things are going well maybe I will finish today otherwise It may take more time.
but I think it will be a great addition to pyccel. currently, @MenoIy  still working on the wrapper. so I will work on that until Karim is done.
what do you think?

Reply:
I just mean get the copy/paste mechanism to work, not necessarily an entire internal library. If you have time then I think that would be good. That way we are sure that the method is sufficiently general.

If you plan to work on the wrapper next then I definitely agree that it would be easier to start that after PR #548 has been merged